### PR TITLE
Debugger: shows error message in console

### PR DESCRIPTION
**Pull request by mabar**
I just don't think there is a reason to not show error message in console. But I may be wrong, let me know if I missed something.

### CODE REVIEW

**Comment by mabar** [comment]
Output looks like this:
```
ERROR: Ello there, I am veri broken app.
Error was logged. Enable debug mode to see more info.
```

Build failure is unrelated to the PR, symplify/ecs seems to be broken.

**Comment by mabar** [comment]
I usually enable debug mode, to see stack trace, etc. Writing into log may fail.

What about `To see more, enable debug mode.` and `To see more, enable debug mode or check log.` in case error was logged?

**Comment by dg** [comment]
If it's been logged, there's no need to do anything.

**Comment by mabar** [comment]
> If it's been logged, there's no need to do anything.
Good point, changed.

With log
```
ERROR: Ello there, I am veri broken app.
Check log to see more info.
```

Without log
```
ERROR: Ello there, I am veri broken app.
Unable to log error. You may try enable debug mode to inspect the problem.
```

**Comment by dg** [comment]
Great!

File: src/Tracy/Debugger/Debugger.php [hunk 1]
```diff
@@ -319,8 +319,12 @@ public static function exceptionHandler(\Throwable $exception): void
 					require self::$errorTemplate ?: __DIR__ . '/assets/error.500.phtml';
 				})(empty($e));
 			} elseif (PHP_SAPI === 'cli') {
-				@fwrite(STDERR, 'ERROR: application encountered an error and can not continue. '
-					. (isset($e) ? "Unable to log error.\n" : "Error was logged.\n")); // @ triggers E_NOTICE when strerr is closed since PHP 7.4
+				// @ triggers E_NOTICE when strerr is closed since PHP 7.4
```
Comment (Ciki) [inline src/Tracy/Debugger/Debugger.php:322]: there's a typo `strerr` should be `stderr`

Comment (dg) [reply src/Tracy/Debugger/Debugger.php:322]: serepes

**Pull request merged**